src/NServiceBus.Metrics/CriticalTime/CriticalTimeMetricBuilder.cs

```diff
@@ -23,6 +35,6 @@ public override void WireUp(FeatureConfigurationContext featureConfigurationCont
         });
     }
 
-    [Timer("Critical Time", "Messages", "Age of the oldest message in the queue.")]
+    [Timer("Critical Time", "Messages", "The time it took from sending to processing the message.")]
```

Comment:
I've tested it so far and it doesn't really seem to matter. That could of course also be a reason not to touch it.